Tests/Functional/ErrorHandling/main.swift

```diff
@@ -104,6 +108,19 @@ class ErrorHandling: XCTestCase {
         XCTAssertEqual(try functionThatShouldReturnButThrows(), 1)
     }
 }
+
+// CHECK: Test Case 'ErrorHandling.test_shouldNotThrowErrorDefiningSuccess' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ErrorHandling.test_shouldNotThrowErrorDefiningSuccess' passed \(\d+\.\d+ seconds\)
+func test_shouldNotThrowErrorDefiningSuccess() {
+    XCTAssertNoThrow(try functionThatDoesNotThrowError())
+}
+
+// CHECK: Test Case 'ErrorHandling.test_shouldThrowErrorDefiningError' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ErrorHandling.test_shouldThrowErrorDefiningError' passed \(\d+\.\d+ seconds\)
```

Comment:
Oh, I'm sorry, I scanned over this and didn't realize this was a failing test. If this test is meant to fail, then these assertions are incorrect, and will fail the swift-corelibs-xctest test suite.

Here, these `CHECK` statements are assertions that the text "Test Case 'ErrorHandling.test_shouldThrowErrorDefiningError' passed" must appear in the test output. Since `test_shouldThrowErrorDefiningError` will fail, this output will definitely not appear in the output.

I'd like for you to add a `CHECK` assertion to verify that `XCTAssertNoThrow(try functionThatDoesThrowError())` will fail a test case. Other tests in this file, such as `test_throwsErrorInAssertionButFailsWhenCheckingError`, are examples of failure assertions.